cmd/traffic_manager/metrics.cc

```diff
@@ -46,6 +46,7 @@ struct Evaluator {
   bool
   bind(lua_State *L, const char *metric, const char *expression)
   {
+    const char *err = nullptr;
```

Comment:
Hmm.... I think it should not move in the block. It may cause error with compiling 
```
crosses initialization of ‘const char* err’
```